src/main/java/org/humancellatlas/ingest/submission/GraphValidationError.java

```diff
@@ -0,0 +1,12 @@
+// Temporary classes until dcp-507 is done
+
+package org.humancellatlas.ingest.submission;
+import lombok.Data;
+
+@Data
+public class GraphValidationError {
+    private String test;
+    private String message;
+    private Object[] affectedEntities;
```

Comment:
could also define classes here

Reply:
I was going to but I have literally removed this in my next PR